docs/src/styles/custom.css

```
@@ -7,276 +7,298 @@
 
 /* ------------------------------------------------------------------
    3. TYPOGRAPHY  (TASK-003)
    System font stack + fluid headings
    ------------------------------------------------------------------ */
 :root {
-  --sl-font: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
-    Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
-  --sl-font-mono: ui-monospace, "Cascadia Code", "Source Code Pro", Menlo,
-    Consolas, "DejaVu Sans Mono", monospace;
+	--sl-font:
+		system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial,
```

Stylelint (17.10.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)